.github/workflows/contracts.yml

```
@@ -3,107 +3,126 @@ name: Contracts CI
-        uses: dtolnay/rust-toolchain@stable
-        with:
-          targets: wasm32v1-none
-
-      - name: Cache Rust artifacts
-        uses: Swatinem/rust-cache@v2
-        with:
-          workspaces: |
-            soroban -> soroban/target
-
+      - "contracts/**"
+name: Contracts CI
```

actionlint (1.7.12)

[error] key "name" is duplicated in "workflow" section. previously defined at line:1,col:1

(syntax-check)